core/src/main/java/io/confluent/kafka/schemaregistry/storage/SubjectContainer.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafka.schemaregistry.storage;
+
+public interface SubjectContainer {
```

Comment:
rather than an interface, would it better to introduce another abstract class extending `SchemaRegistryKey` that takes a subject and provides a getter for it?